client-failover/client-failover-sample-hazelcast5/src/main/java/sample/com/hazelcast/clientfailover/ClientWithSsl.java

```diff
@@ -0,0 +1,127 @@
+ * See: https://docs.hazelcast.com/cloud/get-started
+ * 
+ * The numbers in angled brackets are used in the documentation 
+ * to call out and explain important steps.
+ * 
+ * The comments marked tag:: are also used to include and exclude 
+ * snippets in the docs.
+ */
+// end::intro-comments[]
+
+public class ClientWithSsl {
+
+    public static void main(String[] args) throws Exception {
+
+        ClientFailoverConfig clientFailoverConfig = new ClientFailoverConfig(); // <2>
+        clientFailoverConfig.addClientConfig(getPrimaryClientConfig());
+        clientFailoverConfig.addClientConfig(getSecondaryClientConfig());
+
+        HazelcastInstance client = HazelcastClient.newHazelcastFailoverClient(clientFailoverConfig);
+
+        System.out.println("Connection Successful!");
+
+        nonStopMapExample(client); // <3>
+
+        client.shutdown();
+
+        System.exit(0);
+
+    }
+
+    // <4>
+    private static ClientConfig getPrimaryClientConfig() throws Exception {
+        ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
+        Properties props = new Properties();
+        props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client.keystore").toURI().getPath());
+        props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        props.setProperty("javax.net.ssl.trustStore",
+            classLoader.getResource("client.truststore").toURI().getPath());
+        props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setRedoOperation(true);
+        config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
+        config.getNetworkConfig().getCloudConfig()
+            .setDiscoveryToken("YOUR_DISCOVERY_TOKEN")
+            .setEnabled(true);
+        config.setProperty("hazelcast.client.cloud.url", "YOUR_CLIENT_URL");
+        config.setClusterName("YOUR_CLUSTER_ID");
+        config.getConnectionStrategyConfig()
+            .getConnectionRetryConfig()
+            .setClusterConnectTimeoutMillis(10000); // <5>
+        return config;
+        }
+    
+    // <6>
+    private static ClientConfig getSecondaryClientConfig() throws Exception {
+        ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
+       Properties props = new Properties();
+        props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client2.keystore").toURI().getPath()); // <7>
+        props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        props.setProperty("javax.net.ssl.trustStore",
+            classLoader.getResource("client2.truststore").toURI().getPath()); // <7>
+        props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setRedoOperation(true);
```

Comment:
We don't need this with exception handling.

Reply:
Updated as requested.